stdpopsim/models.py

```diff
@@ -79,20 +82,22 @@ def __init__(
         long_description,
         generation_time=None,
         citations=None,
-        qc_model=None,
+        qc=None,
         population_configurations=None,
         demographic_events=None,
         migration_matrix=None,
         population_id_map=None,
         populations=None,
         model=None,
+        mutation_rate=None,
     ):
         self.id = id
         self.description = description
         self.long_description = long_description
         self.generation_time = 1 if generation_time is None else generation_time
+        self.mutation_rate = mutation_rate
         self.citations = [] if citations is None else citations
-        self.qc_model = qc_model
```

Comment:
What's the reasoning for this change?

Reply:
Previously, the QC model (just the `Demography` object) was attached to a model to check 1) that a model has been QC'd (is not None), and 2) to check for equivalency in tests. I think it would also be good to check equivalency of other attributes of the model to its QC counterpart, such as generation time and mutation rate, so that those are more easily available to compare to each other if the full QC stdpopsim DemographicModel object is attached instead.

Reply:
Previously, the QC model (just the Demography object) was attached to a model to check 1) that a model has been QC'd (is not None), and 2) to check for equivalency in tests. I think it would also be good to check equivalency of other attributes of the model to its QC counterpart, such as generation time and mutation rate, so that those are more easily available to compare to each other if the full QC stdpopsim DemographicModel object is attached instead.

Reply:
hm, but this is changing the API for the Demographic Model - if we really want to change this, we should probably leave `qc_model` as a deprecated synonym... but, this *is* a "model" after all, so `qc_model` doesn't seem so bad?

Reply:
That's true - I'm fine with either name, and that avoids a param name change. In the equivalency tests we would then check `DemographicModel.model == DemographicModel.qc_model.model`

Reply:
yeah i agree with Peter here. I think we should probably avoid changing the param name